.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.13.0
+    hooks:
+    -   id: pyupgrade
```

Comment:
Doesn't ruff also replace pyupgrade and isort? https://github.com/astral-sh/ruff#rules

Reply:
Good point. I'll revise.